lib/server/index.js

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
As I understand it from reading through this, the `./lib` directory is for code shared between performance-tests and integration-tests?  Perhaps `test-utils` which clearly marks this code as non-production, rather than the amibiguous `lib`?